EasyPost/Services/ApiKeyService.cs

```diff
@@ -33,6 +37,35 @@ internal ApiKeyService(EasyPostClient client)
         [CrudOperations.Read]
         public async Task<ApiKeyCollection> All(CancellationToken cancellationToken = default) => await RequestAsync<ApiKeyCollection>(Method.Get, "api_keys", cancellationToken);
 
+        /// <summary>
+        ///     Retrieve the <see cref="ApiKey"/>s for a specific <see cref="User"/>.
+        /// </summary>
+        /// <param name="id">The ID of the <see cref="User"/> to retrieve keys for.</param>
+        /// <param name="cancellationToken"><see cref="CancellationToken"/> to use for the HTTP request.</param>
+        /// <returns>A list of <see cref="ApiKey"/>s for the specified <see cref="User"/>.</returns>
+        /// <exception cref="FilteringError">Thrown if the specified <see cref="User"/> does not exist.</exception>
+        public async Task<List<ApiKey>?> RetrieveApiKeysFor(string id, CancellationToken cancellationToken = default)
```

Comment:
We should name this function as `api_keys()` to match other CLs

Reply:
Resolved per our internal conversations.